src/tools/pkcs11-tool.c

```diff
@@ -4755,7 +4790,12 @@ derive_ec_key(CK_SESSION_HANDLE session, CK_OBJECT_HANDLE key, CK_MECHANISM_TYPE
 	if (BIO_read_filename(bio_in, opt_input) <= 0)
 		util_fatal("Cannot open %s: %m", opt_input);
 
+#if OPENSSL_VERSION_NUMBER >= 0x30200000L
+/* TODO following is in OpenSSL master not  3.0.8 */
```

Comment:
please, add a link to opensc commit or pull request implementing this (if you find it) so it is easy to recheck and update the condition accordingly when it will be in some released version.

Reply:
Comments added https://github.com/OpenSC/OpenSC/pull/2715/commits/f216aa86fe3315d8cb7d97e3709655581cde19cc
Fix is in master, should be in 3.2.0.